dev/vision-n-scope/7-contributions.md

```diff
@@ -12,7 +12,40 @@ placeholder
 
 ## Carter MacNab
 
-placeholder
+### Introduction
+
+ By the end of Capstone, I'd like to have worked on and completed the player client and scoreboard software necessary for the functionality of our project. I also want to play a role in observing and implementing rigid security rules to ensure our project stays free of potential vulnerability concerns. Developing the player client and focusing on security concerns will allow me to play a role in advising and learning from my fellow teammates during the course of development. 
+### System Architecture
```

Comment:
i would get rid of the section headers and just add your comments in another paragraph in the introduction. since the system architecture is more or less global to our individual sections, its more clutter to add it every time